pkg/asset/templates/kubeconfig.yaml

```diff
@@ -6,10 +6,11 @@ clusters:
     server: {{ .Server }}
     certificate-authority-data: {{ .CACert }}
```

Comment:
To be clear, two separate certificates are generated and need to be placed on host via this kubeconfig (or in the automated situation by using this generated kubeconfig file directly).